wemake_python_styleguide/logic/tree/classes.py

```diff
@@ -34,3 +39,54 @@ def is_forbidden_super_class(class_name: Optional[str]) -> bool:
     if class_name in ALLOWED_BUILTIN_CLASSES:
         return False
     return is_builtin_name(class_name)
+
+
+def get_attributes(
+    node: ast.ClassDef,
+) -> Tuple[List[types.AnyAssign], List[ast.Attribute]]:
+    """Returns all class and instance attributes of a class."""
+    class_attributes = []
+    instance_attributes = []
+
+    for nd in ast.walk(node):
```

Comment:
Maybe nd -> subnode for consistency

Reply:
Will fix

Reply:
It seems to have been shortened to pass the linters line too long violation.